app/assets/stylesheets/external-links.scss

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I don't think we even need this file at all. You should use flexbox helpers whenever possible.
See: https://bulma.io/documentation/helpers/flexbox-helpers/

Reply:
I've removed everything except `fill: #4a4a4a;` From what I saw in Bulma docs, color and icon helpers are modifying `color` and we need to modify `fill`.

As a solution to eliminate the file, I can add a new class eg `fill-gray` to `application.scss` and assign icons to it. Would that be ok?

Reply:
It might work even if you just change the color. (Maybe bulma sets `fill: currentColor`, which picks up the color from the `color` property.)

Reply:
I've tried that and it doesn't seem to work. But I noticed that it works for map icon in ad details. Took me some time but I've figured out that this is because I'm using fontawesome solid icons which have no stroke color but fill the inside of the shape.

I'll check out how regular icons look and post a screenshot here shortly

Reply:
If you feel adventurous, you could try adding `fill="currentColor"` in the svg files in the `<path>` tag. :)

Reply:
That worked like a charm! Good tip 🦑

Reply:
Out of curiosity, have you tried adding a text colour class to

Reply:
Yup. Didn't work.